test/unit/list/containers/no-matching-placeholder-test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import chai, { expect } from "chai";
+import chaiEnzyme from "chai-enzyme";
+import React from "react";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import thunk from "redux-thunk";
+
+import mountWithL10n from "test/unit/mocks/l10n";
+import { initialState } from "../manage/mock-redux-state";
+import NoMatchingPlaceholder from "src/list/containers/no-matching-placeholder";
+
+chai.use(chaiEnzyme());
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+// TODO: figure out how to properly test with chai + enzyme + fluent + react overlays
+//    so that openFAQ dispatching can be verified
```

Comment:
I poked at this and wasn't able to quite get it wired up, either. I think the answer is going to involve a [custom markup parser](https://github.com/projectfluent/fluent.js/wiki/React-Overlays#custom-markup-parsers).

Reply:
I'll try to play around with it; see if React.render gets us over the line

Reply:
I tried some options using the custom markup parser.  Short of reimplementing the hooks in the component under test (which defeats the purpose of testing), I'm not able to get the desired results. 😢